src/toolbar/sticky/stickytoolbarview.js

```diff
@@ -20,21 +20,88 @@ export default class StickyToolbarView extends ToolbarView {
 	/**
 	 * @inheritDoc
```

Comment:
The limiter has been moved to the model. There's no need for an extra argument anymore.